File: sdkwork-backend-sdk-go/types/plus_api_key_vo.go
package types

// API密钥VO类，用于传输系统中API访问凭证信息
type PlusApiKeyVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	Name string `json:"name"`
	KeyValue string `json:"keyValue"`
	MaskedKeyValue string `json:"maskedKeyValue"`
	Type string `json:"type"`
	Status string `json:"status"`
	ExpireTime string `json:"expireTime"`
	Description string `json:"description"`
	LastUsedTime string `json:"lastUsedTime"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusApiKeyVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusApiKeyVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public string? Name { get; set; }
        public string? KeyValue { get; set; }
        public string? MaskedKeyValue { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public string? ExpireTime { get; set; }
        public string? Description { get; set; }
        public string? LastUsedTime { get; set; }
    }
}
